docs/developer-guide/syntax-reference.rst

```diff
@@ -839,7 +839,8 @@ In the above statements from_item is one of the following:
 -  ``from_item LEFT JOIN from_item ON join_condition``
 
 The WHERE clause can refer to any column defined for a stream or table,
-including the two implicit columns ``ROWTIME`` and ``ROWKEY``.
+including the two implicit columns ``ROWTIME`` and ``ROWKEY``. If these
```

Comment:
You don't need to mention this. They can be used as other columns in all operators including BETWEEN.

Reply:
Got it - I will add a dedicated section as discussed offline about BETWEEN next to LIKE.